state-chain/node/src/chain_spec.rs

```diff
@@ -787,6 +787,8 @@ fn testnet_genesis(
 			witness_safety_margin: Some(bitcoin_safety_margin),
 			dust_limits: vec![(btc::Asset::Btc, BITCOIN_DUST_LIMIT)],
 			max_swap_retry_duration_blocks: 600,
+			max_dca_chunks: 50,
```

Comment:
As discussed, doesn't the deafult value for the pallets config apply here (which sets these to reasonable values)? If so, we can just use `Default::default()` as below.